docs/index.rst

```diff
@@ -65,11 +63,11 @@ package for plasma physics currently under development.
 
    Vision Statement <about/vision_statement>
    Installing <install>
-   auto_examples/index
    COMMUNICATION
    CONTRIBUTING
    CODE_OF_CONDUCT
    about/citation
+   examples
```

Comment:
I think it makes more sense for **Examples** to follow **Installing**, but I can see arguments for having it at the bottom of the stack too.